packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -50,17 +55,28 @@ interface Props {
 const AccessibilityRating = ({
   gradationMap,
   grayscale = false,
+  intl,
```

Comment:
Instead of passing `intl` as a prop, use the hook `useIntl` to access the `intl` context from within the component without the need for a breaking change.